examples/jvm-interop/bridge.c

```diff
@@ -0,0 +1,248 @@
+#include <errno.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stddef.h>
+#include <string.h>
+#include <unistd.h>
+#include <stdalign.h>
+#include <stdint.h>
+
+#include <jni.h>
+#include "HelloJNI.h"
+
+
+void *roc_alloc(size_t size, unsigned int alignment)
+{
+    return malloc(size);
+}
+
+void *roc_realloc(void *ptr, size_t new_size, size_t old_size,
+                  unsigned int alignment)
+{
+    return realloc(ptr, new_size);
+}
+
+void roc_dealloc(void *ptr, unsigned int alignment)
+{
+   free(ptr);
+}
+
+__attribute__((noreturn)) void roc_panic(void *ptr, unsigned int alignment)
+{
+   // TODO
```

Comment:
🎨 No need to change, but it might help future contributors to note what should happen here instead (e.g. `// TODO throw a RuntimeException from C`)

Reply:
added, thanks!